### PR TITLE
Revamp Kubernetes messaging in 'Get Started' / Quickstart

### PR DESCRIPTION
**Pull request by metral**
Ref: https://github.com/pulumi/docs/issues/1045.

It's best to pull down and run this PR locally to examine the breadth of the updated docs.

### CODE REVIEW

File: content/quickstart/kubernetes/faq.md [hunk 1]
```diff
@@ -1,120 +1,176 @@
...
+
+The service is used to coordinate multiple user updates, so they don't update the
+same resources concurrently, and stores the state history of your deployments.
+
+If you ask Pulumi to boot up a cluster (_e.g._, as our [EKS example][eks] shows), then it is
+possible to retrieve a kubeconfig file _for that cluster only_. In this case, the kubeconfig file is
+stored as part of the state history of rollouts -- but they are considered opaque state by the
+Pulumi service. The service **never** uses these values for any reason whatsoever.
+
+More over, the contents of the kubeconfig files are not sensitive for managed clusters
```
Comment (lblackstone) [inline content/quickstart/kubernetes/faq.md]:
```suggestion
Additionally, the contents of the kubeconfig files are not sensitive for managed clusters
```

File: content/quickstart/kubernetes/faq.md [hunk 2]
```diff
@@ -1,120 +1,176 @@
...
+on the major cloud providers, as they defer to their authentication mechanisms for their respective CLI.
```
Comment (lblackstone) [inline content/quickstart/kubernetes/faq.md]:
```suggestion
on the major cloud providers, as they defer authentication to their local CLIs.
```

File: content/quickstart/kubernetes/faq.md [hunk 3]
```diff
@@ -1,120 +1,176 @@
...
+
+## Does the Pulumi Service see my cloud provider credentials?
+
+No. The `pulumi` CLI tool runs locally in the user's client. This allows
+Pulumi to transparently rely on the underlying cloud provider's CLI and authentication
+mechanisms. The service does not read, use, depend, or store any credentials.
```
Comment (lblackstone) [inline content/quickstart/kubernetes/faq.md]:
```suggestion
mechanisms. The service does not read, use, depend on, or store any credentials.
```

File: content/quickstart/kubernetes/faq.md [hunk 4]
```diff
@@ -1,120 +1,176 @@
...
+
+## Can I use Pulumi without the pulumi.com service?
+
+Yes. The service performs several functions that are useful, such as coordinating so that
+multiple users don't simultaneously update the same resources, state
+checkpointing & management, and team auth/identity.
```
Comment (lblackstone) [inline content/quickstart/kubernetes/faq.md]: I think there are a few other notable features. Wouldn't hurt to include a comprehensive list from https://www.pulumi.com/pricing/

File: content/quickstart/kubernetes/introduction.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+to connect to it, the Pulumi CLI should "just work" transparently out of the box
+after importing the [`pulumi/kubernetes`][pulumi-k8s] package into your program.
+
+Pulumi supports any Kubernetes version that is currently supported by 
+Kubernetes [upstream][upstream].
+
+## Why Pulumi?
+
+In addition to being compatible with all existing Kubernetes workflows, Pulumi provides several
+advantages:
```
Comment (lblackstone) [inline content/quickstart/kubernetes/introduction.md]: Probably worth linking to https://blog.pulumi.com/pulumi-a-better-way-to-kubernetes

File: content/quickstart/kubernetes/tutorial-clusters.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+title: Tutorials - Clusters
+aliases: ["/reference/kubernetes-clusters.html"]
+menu:
+  quickstart:
+    identifier: kubernetes-clusters
+    weight: 6
+    parent: kubernetes
+---
+
+<img src="/images/quickstart/k8s-purple.png" align="right">
```
Comment (lblackstone) [inline content/quickstart/kubernetes/tutorial-clusters.md]: I don't think the k8s logo is allowed to be purple

File: content/quickstart/kubernetes/tutorial-workloads.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+title: Tutorials - Workloads
+aliases: ["/reference/kubernetes-workloads.html"]
+menu:
+  quickstart:
+    identifier: kubernetes-workloads
+    weight: 6
+    parent: kubernetes
+---
+
+<img src="/images/quickstart/k8s-purple.png" align="right">
```
Comment (lblackstone) [inline content/quickstart/kubernetes/tutorial-workloads.md]: Don't think the logo can be purple

File: content/quickstart/kubernetes/tutorial-azure-kubernetes-service.md [hunk 8]
```diff
@@ -1,9 +1,13 @@
 ---
-title: "Tutorial: Azure Kubernetes Service"
-aliases: ["tutorial-azure-kubernetes-service.html"]
+title: "Tutorial: AKS - Hello World!"
+aliases:
+    - tutorial-aks.html
+    - /quickstart/kubernetes/tutorial-azure-kubernetes-service
```
Comment (justinvp) [inline content/quickstart/kubernetes/tutorial-azure-kubernetes-service.md]:
```suggestion
    - /quickstart/azure/tutorial-azure-kubernetes-service.html
    - /quickstart/azure/tutorial-azure-kubernetes-service/
```

There are two aliases (redirects) needed for this file:

1. One for the old pre-Hugo ugly style URL: `/quickstart/azure/tutorial-azure-kubernetes-service.html`
2. One for the Hugo pretty style URL at the previous location: `/quickstart/azure/tutorial-azure-kubernetes-service/`

File: content/quickstart/kubernetes/tutorial-eks.md [hunk 9]
```diff
@@ -2,10 +2,11 @@
 title: "Tutorial: EKS - Hello World!"
 aliases:
     - tutorial-eks.html
```
Comment (justinvp) [inline content/quickstart/kubernetes/tutorial-eks.md]:
```suggestion
    - /quickstart/aws/tutorial-eks.html
```

File: content/quickstart/kubernetes/tutorial-eks.md [hunk 10]
```diff
@@ -2,10 +2,11 @@
...
-    - /quickstart/eks-hello-world.html
+    - /quickstart/kubernetes/eks-hello-world.html
```
Comment (justinvp) [inline content/quickstart/kubernetes/tutorial-eks.md]:
```suggestion
    - /quickstart/aws/tutorial-eks/
    - /quickstart/eks-hello-world.html
```

The `/quickstart/eks-hello-world.html` alias should remain. And one should be added for the old pretty-style URL location: `/quickstart/aws/tutorial-eks/`.

Altogether the aliases for this file should be:

```
aliases:
    - /quickstart/eks-hello-world.html
    - /quickstart/aws/tutorial-eks.html
    - /quickstart/aws/tutorial-eks/
```

File: content/quickstart/kubernetes/tutorial-gke.md [hunk 11]
```diff
@@ -2,10 +2,11 @@
 title: "Tutorial: GKE - Hello World!"
 aliases:
     - tutorial-gke.html
```
Comment (justinvp) [inline content/quickstart/kubernetes/tutorial-gke.md]:
```suggestion
    - /quickstart/gcp/tutorial-gke.html
```

File: content/quickstart/kubernetes/tutorial-gke.md [hunk 12]
```diff
@@ -2,10 +2,11 @@
...
-    - /quickstart/gke-hello-world.html
+    - /quickstart/kubernetes/gke-hello-world.html
```
Comment (justinvp) [inline content/quickstart/kubernetes/tutorial-gke.md]:
```suggestion
    - /quickstart/gcp/tutorial-gke/
    - /quickstart/gke-hello-world.html
```

Altogether the aliases for this file should be:

```
aliases:
    - /quickstart/gke-hello-world.html
    - /quickstart/gcp/tutorial-gke.html
    - /quickstart/gcp/tutorial-gke/
```

File: content/quickstart/kubernetes/tutorial-workloads.md [hunk 13]
```diff
@@ -0,0 +1,13 @@
+---
+title: Tutorials - Workloads
+aliases: ["/reference/kubernetes-workloads.html"]
```
Comment (justinvp) [inline content/quickstart/kubernetes/tutorial-workloads.md]:
```suggestion
```

I don't think this alias (redirect) is needed. Did we previously have a `/reference/kubernetes-workloads.html` file?

File: content/quickstart/kubernetes/tutorial-clusters.md (hunk 6 above)
Comment (justinvp) [reply content/quickstart/kubernetes/tutorial-clusters.md]: This is an existing image we use elsewhere. We can fix this as part of #1107.

**Comment by metral** [comment]
@lblackstone @justinvp Feedback has been addressed. PTAL

@joeduffy @lukehoban any thoughts before merging?

File: content/reference/cli/pulumi_login.md [hunk 14]
```diff
@@ -17,25 +17,25 @@ The service manages your stack's state reliably. Simply run
     $ pulumi login
 
 and this command will prompt you for an access token, including a way to launch your web browser to
-easily obtain one. You can script by using PULUMI_ACCESS_TOKEN environment variable.
+easily obtain one. You can script by using `PULUMI_ACCESS_TOKEN` environment variable.
```
Comment (justinvp) [inline content/reference/cli/pulumi_login.md]: Just noticed that this is a change to a generated file. It'd be best to just revert the changes to this file (since they'll be clobbered over the next time we regenerate), and instead make any improvements to the source: https://github.com/pulumi/pulumi/blob/8e0da42412dd3140782446db874ab7a73ec5cda1/cmd/login.go#L40

Comment (metral) [reply content/reference/cli/pulumi_login.md]: Great catch! This is being tracked in https://github.com/pulumi/pulumi/pull/2810. I'll go ahead and remove the changes to `pulumi login` from this PR
